apidoc/Titanium/UI/ListView.yml

```diff
@@ -849,6 +849,9 @@ properties:
 
   - name: separatorInsets
     summary: The insets for list view separators (applies to all cells). This property is applicable on iOS 7 and greater.
+    deprecated:
+        since: "5.2.0"
```

Comment:
The [Ti.UI.TableView version of this PR](https://github.com/appcelerator/titanium_mobile/blob/master/apidoc/Titanium/UI/TableView.yml#L1553) says deprecated since 5.4.0, this one since 5.2.0. They should be the same